http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/BucketHelper.scala

```diff
@@ -142,13 +145,44 @@ class BucketHelper[F[_]: Concurrent: ContextShift: Parallel](config: BucketHelpe
           )).compile.drain
         }
 
+    val uploadGpuDockerCompose = gpuConfig.traverse { gC =>
+      val additionalGpuConfigString = (1 to gC.numOfGpus)
```

Comment:
Thought about using `circe-yaml` to modify the docker-compose file but I guess this is fine 🙈

Reply:
ah good to know....maybe worth using `circe-yaml` if the use case becomes more complicated